DESCRIPTION

```diff
@@ -35,6 +35,7 @@ Imports:
     forcats,
     tidytext,
     textdata,
-    dplyr
+    dplyr,
+    magrittr
```

Comment:
did you use this package? i can't see where you used it

Reply:
I think it's for pipes? I just tried to push and am trying to resolve merge conflict now.

Reply:
okay sounds good